backend/app/models.py

```diff
@@ -163,11 +163,19 @@ class GEOIDS(BaseModel):
     geoids: list[str]
 
 
+class ZoneAndGEOIDS(GEOIDS):
```

Comment:
O: Naming? I guess this is explicit...! I would have maybe called this AssignedGEOIDS or something 🤷🏻

Reply:
Resolved in defeb6ab38d32bcaa8a1863526e3e5a32334ee7e